monkey_island/cc/resources/island_logs.py

```diff
@@ -0,0 +1,19 @@
+import flask_restful
+import logging
+
+from cc.auth import jwt_required
+from cc.services.island_logs import IslandLogService
+
+__author__ = "Maor.Rayzin"
+
+logger = logging.getLogger(__name__)
+
+
+class IslandLog(flask_restful.Resource):
+    @jwt_required()
+    def get(self):
+        try:
+            return IslandLogService.get_log_file()
+        except Exception as e:
+            logger.error('Monkey Island logs failed to download', exc_info=True)
```

Comment:
I'd rather in this case return some sort of error message to the user, maybe indicating the exception, because this is a rare case where you can't debug the problem! 🤕

Reply:
Ok good idea